### PR TITLE
docs: update github pages links

### PR DESCRIPTION
**Pull request by tskxz**
As referred to in this issue #2967 I updated the links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ youki is named after the Japanese word 'youki', which means 'a container'. In Ja
 > $ sudo podman run --cgroup-manager=cgroupfs --runtime /workspaces/youki/youki hello-world
 > ```
 
-[User Documentation](https://containers.github.io/youki/user/basic_setup.html#quick-install)
+[User Documentation](https://youki-dev.github.io/youki//user/basic_setup.html#quick-install)
```
Comment (TBBle) [inline README.md:32]:
```suggestion
[User Documentation](https://youki-dev.github.io/youki/user/basic_setup.html#quick-install)
```

File: README.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ We have [our roadmap](https://github.com/orgs/containers/projects/15).
 
 # 🎨 Design and implementation of youki
 
-The User and Developer Documentation for youki is hosted at [https://containers.github.io/youki/](https://containers.github.io/youki/)
+The User and Developer Documentation for youki is hosted at [https://youki-dev.github.io/youki//](https://youki-dev.github.io/youki//)
```
Comment (TBBle) [inline README.md:99]:
```suggestion
The User and Developer Documentation for youki is hosted at [https://youki-dev.github.io/youki/](https://youki-dev.github.io/youki/)
```

File: README.md [hunk 3]
```diff
@@ -277,7 +277,7 @@ just youki-dev # or youki-release
 
 # 👥 Community and Contributing
 
-Please refer to [our community page](https://containers.github.io/youki/community/introduction.html).
+Please refer to [our community page](https://youki-dev.github.io/youki//community/introduction.html).
```
Comment (TBBle) [inline README.md:280]:
```suggestion
Please refer to [our community page](https://youki-dev.github.io/youki/community/introduction.html).
```

File: docs/src/developer/documentation_mdbook.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ This documentation is created using mdbook and aims to provide a concise referen
...
 
-Currently this documentation is hosted at [https://containers.github.io/youki/](https://containers.github.io/youki/), using GitHub pages. GitHub CI actions are used to automatically check if any files are changed in /docs on each push / PR merge to main branch, and if there are any changes, the mdbook is build and deployed to gh-pages. We use [https://github.com/peaceiris/actions-mdbook](https://github.com/peaceiris/actions-mdbook) to build and then [https://github.com/peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) GitHub action to deploy the mdbook.
+Currently this documentation is hosted at [https://youki-dev.github.io/youki//](https://youki-dev.github.io/youki//), using GitHub pages. GitHub CI actions are used to automatically check if any files are changed in /docs on each push / PR merge to main branch, and if there are any changes, the mdbook is build and deployed to gh-pages. We use [https://github.com/peaceiris/actions-mdbook](https://github.com/peaceiris/actions-mdbook) to build and then [https://github.com/peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) GitHub action to deploy the mdbook.
```
Comment (TBBle) [inline docs/src/developer/documentation_mdbook.md:7]:
```suggestion
Currently this documentation is hosted at [https://youki-dev.github.io/youki/](https://youki-dev.github.io/youki/), using GitHub pages. GitHub CI actions are used to automatically check if any files are changed in /docs on each push / PR merge to main branch, and if there are any changes, the mdbook is build and deployed to gh-pages. We use [https://github.com/peaceiris/actions-mdbook](https://github.com/peaceiris/actions-mdbook) to build and then [https://github.com/peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) GitHub action to deploy the mdbook.
```

File: tools/wasm-sample/README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ This is a simple wasm module for testing purposes. It prints out the arguments g
 cargo build --target wasm32-wasi
 ```
 
-If you want youki to execute the module you must copy it to the root file system of the container and reference it in the args of the config.json. You must also ensure that the annotations contain `"run.oci.handler": "wasm"` and that youki has been compiled with one of the supported wasm runtimes. For further information please check the [documentation](https://containers.github.io/youki/user/webassembly.html).
+If you want youki to execute the module you must copy it to the root file system of the container and reference it in the args of the config.json. You must also ensure that the annotations contain `"run.oci.handler": "wasm"` and that youki has been compiled with one of the supported wasm runtimes. For further information please check the [documentation](https://youki-dev.github.io/youki//user/webassembly.html).
```
Comment (TBBle) [inline tools/wasm-sample/README.md:7]:
```suggestion
If you want youki to execute the module you must copy it to the root file system of the container and reference it in the args of the config.json. You must also ensure that the annotations contain `"run.oci.handler": "wasm"` and that youki has been compiled with one of the supported wasm runtimes. For further information please check the [documentation](https://youki-dev.github.io/youki/user/webassembly.html).
```

**Review by TBBle — CHANGES_REQUESTED** [review]
I had a look, and a bunch of places now have a double-/ in the page URL. I guess you did a search-replace from `containers.github.io/youki` to `youki-dev.github.io/youki/` and now there's a few URLs like `https://youki-dev.github.io/youki//community/introduction.html` in the docs.

URLs that previously ended with `youki` have become URLs that end with `youki/`, and _that_ change looks correct to me (the site redirects to have the trailing `/` in this case) so it's only the double-`//` that need to be fixed.

**Comment by tskxz** [comment]
> I had a look, and a bunch of places now have a double-/ in the page URL. I guess you did a search-replace from `containers.github.io/youki` to `youki-dev.github.io/youki/` and now there's a few URLs like `https://youki-dev.github.io/youki//community/introduction.html` in the docs.
> 
> URLs that previously ended with `youki` have become URLs that end with `youki/`, and _that_ change looks correct to me (the site redirects to have the trailing `/` in this case) so it's only the double-`//` that need to be fixed.

Thanks for noticing it, I think now it should be fine and fixed.

**Review by TBBle — APPROVED** [review]

**Review by utam0k — APPROVED** [review]

**Comment by utam0k** [comment]
Thanks!

**Pull request merged**